Components/src/commonMain/kotlin/alerts/Alerts.kt

```diff
@@ -0,0 +1,53 @@
+package com.splendo.kaluga.alerts
+
+/*
+
+Copyright 2019 Splendo Consulting B.V. The Netherlands
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+*/
+
+data class Alert(
+    val identifier: String,
+    val title: String?,
+    val message: String?,
+    val style: Style = Style.ALERT,
+    val actions: List<Action>
```

Comment:
Should we force users to have at least one action for the Alert? cc @thoutbeckers

Reply:
If so I would make a separate parameter for it, but you could also use a default "ok" or something if there is nothing set.